monero-GUI-guide.md

```diff
@@ -245,7 +246,11 @@ The `Seeds & Keys` tab displays your wallets mnemonic seed as well as your secre
 **(6) Export Spendable Wallet: DO NOT share your spendable wallet qr code with anyone. This can be used like a mnemonic seed for recovering your wallet.** This creates a qrcode that contains all of your keys.    
 **(7) Export View Only Wallet:** This creates a qr code that contains only the keys for viewing the transactions that this wallet sends or receives, but cannot create transactions.    
 
-# 7. Common issues and solutions
+# 7. Binaries Verification
+
+Verify that the files you downloaded match the official ones, you can use [This step-by-step guide](https://getmonero.org/resources/user-guides/verification-windows-beginner.html) with pictures (easy, For windows user)
```

Comment:
Just some grammatical fixes.

`Verify that the files you downloaded match the official ones, you can use [this step-by-step guide](https://getmonero.org/resources/user-guides/verification-windows-beginner.html) with pictures (easy, for Windows users).`

Reply:
edited